### PR TITLE
fix: broken infinite scroll artist artwork grid

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [DIA-63] 

### Description

This PR fixes the broken infinite scroll artwork grid in android.

https://github.com/artsy/eigen/assets/21178754/14475d80-aabd-41ff-bb7a-fdfe92f854b7

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

- broken infinite scroll artist artwork grid

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-63]: https://artsyproduct.atlassian.net/browse/DIA-63?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Android user-facing changes (broken infinite scroll artist artwork grid)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 35affd7a268ecc6a5a8bd4169a6c39943d8a5402
</p>

File: src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx [hunk 1]
```diff
@@ -35,11 +35,10 @@ const ArtworksGrid: React.FC<ArtworksGridProps> = ({ artist, relay, ...props })
     useShowArtworksFilterModal({ artist })
 
   return (
-    <Tabs.ScrollView keyboardShouldPersistTaps="handled">
+    <TabsFlatList keyboardShouldPersistTaps="handled">
```
Comment (araujobarret) [inline src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx:38]: Can't we use just the Tabs.Flatlist?

This issue with a List inside ScrollView again :(

**Review by araujobarret — APPROVED** [review]
That was fast, thanks!

File: src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx:38]: Nnfortunately not. We will follow up with a fix for all the problematic surfaces. But this needs to go out now due to https://artsy.slack.com/archives/C03N12SR0RK/p1689604509994779?thread_ts=1688839201.702459&cid=C03N12SR0RK

File: src/app/Components/TabsFlatlist.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    FlatListProps<any>,
+    "onScroll" | "data" | "scrollEventThrottle" | "ListHeaderComponent" | "renderItem"
+  > {
+  data: any[]
+  paddingHorizontal?: number
+  innerRef?: React.MutableRefObject<{ getNode(): FlatList<any> } | null>
+}
+
+/**
+ * @deprecated
```
Comment (gkartalis) [inline src/app/Components/TabsFlatlist.tsx:15]: its not actually deprecated, but I would refrain from using in other surfaces. We will followup with a fix

**Pull request merged**